src/main.cpp

```diff
@@ -529,26 +547,20 @@ With no FILE, start recording the current screen.)");
   -m, --muxer               Set the output format to a specific muxer instead of detecting it
                             from the filename.
 
-  -x, --pixel-format        Set the output pixel format. These can be found by running:
-                            *ffmpeg -pix_fmts*
-
   -g, --geometry            Selects a specific part of the screen.
 
   -h, --help                Prints this help screen.
 
   -l, --log                 Generates a log on the current terminal. Debug purposes.
 
-  -o, --output              Specify the output where the video is to be recorded.
-
-  -p, --codec-param         Change the codec parameters.
-                            -p <option_name>=<option_value>)");
+  -o, --output              Specify the output where the video is to be recorded.");
```

Comment:
You're missing a closing parenthesis. Not you change `)");` to `");`

Reply:
Thank you so much! That tiny error was driving me crazy and I had no idea what had changed to make the #endif statement no longer be valid.